sympy/integrals/tests/test_integrals.py

```diff
@@ -1060,3 +1060,12 @@ def test_issue_2708():
     integral_f = NonElementaryIntegral(f, (z, 2, 3))
     assert Integral(f, (z, 2, 3)).doit() == integral_f
     assert integrate(f + exp(z), (z, 2, 3)) == integral_f - exp(2) + exp(3)
+
+
+def test_issue_7827():
+    x, n, N = symbols('x n N')
+    assert integrate(summation(x*n, (n, 1, N)), x) == x**2*(N**2/4 + N/4)
+    assert integrate(summation(x*sin(n), (n,1,N)), x) == \
+        Sum(x**2*sin(n)/2, (n, 1, N))
+    assert str(integrate(summation(sin(n*x), (n,1,N)), x)) == \
```

Comment:
Please take a look at #8929 . It covers all these issues. This branch had merge conflicts so opened a new PR.